dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdMappingFileUploadTask.kt

```diff
@@ -66,6 +75,18 @@ open class DdMappingFileUploadTask : DefaultTask() {
     @get:Input
     var mappingFilePath: String = ""
 
+    /**
+     * The sourceSet root folders.
+     */
+    @get:InputFiles
+    var sourceSetRoots: List<File> = emptyList()
+
+    /**
+     * The file containing the repository description.
+     */
+    @get:OutputFile
+    var repositoryFile: File = File("")
```

Comment:
why do you want to keep a reference to this file and not just return it as an optional from `generateRepositoryFile` function